apps/files/src/components/Collaborators/NewCollaborator.vue

```diff
@@ -21,39 +21,30 @@
         </template>
       </oc-autocomplete>
     </oc-grid>
-    <oc-grid gutter="small">
-      <div
-        v-if="selectedCollaborators.length > 0"
-      >
+    <oc-grid gutter="small" v-if="selectedCollaborators.length > 0">
+      <div>
         <div>
-          <div>
-            <translate>Selected collaborators</translate>:
-          </div>
-          <div
-            v-for="(collaborator, index) in selectedCollaborators"
-            :key="collaborator.label"
-            class="uk-flex-inline uk-flex-row uk-flex-start uk-margin-small-bottom"
-            :class="{
-              'uk-margin-small-right': index + 1 !== selectedCollaborators.length
-            }"
-          >
-            <div class="uk-margin-small-top">
-              <span class="uk-text-bold" v-text="collaborator.label" />
-              <span v-if="collaborator.value.shareType === 1" class="uk-text-meta">
-                (<translate>group</translate>)
-              </span>
-            </div>
-            <oc-button :ariaLabel="$gettext('Delete share')" variation="raw" class="oc-cursor-pointer"  @click="$_ocCollaborators_removeFromSelection(collaborator)">
-              <oc-icon
-                name="close"
-                variation="danger"
-              />
-            </oc-button>
-          </div>
+          <translate>Selected collaborators:</translate>
         </div>
+        <template v-for="collaborator in selectedCollaborators">
+          <oc-table middle :key="collaborator.value.shareWith + '-' + collaborator.value.shareType"
```

Comment:
Can we get selected collaborators into one table pls? It's better for accessibility. Also, why is it wrapped in the template? Is there some issue with using `v-for` directly in e. g. table tor?

Reply:
Erm... kind of ridiculous that I didn't use a single table in the first place. Thanks for pointing it out. :-) I am now using a single table and have the `v-for` on the `tr`s, like it should be. 😁

Reply:
Thanks a lot! 🚀